oauth2client/contrib/_metadata.py

```diff
@@ -29,8 +29,13 @@
 from oauth2client import transport
 
 
+
+_GCE_METADATA_HOST = os.getenv('GCE_METADATA_HOST', os.getenv(
```

Comment:
This will always try to find GCE_METADATA_ROOT, which is not a big deal, but can we check for GCE_METADATA_HOST first, like it is done in google-auth https://github.com/googleapis/google-auth-library-python/blob/e756f08dc78616040ab8fbd7db20903137ccf0c7/google/auth/compute_engine/_metadata.py#L39?

Reply:
Sure!